src/main/kotlin/no/nav/faktureringskomponenten/controller/FakturaserieController.kt

```diff
@@ -61,6 +63,26 @@ class FakturaserieController @Autowired constructor(
         return ResponseEntity.ok(NyFakturaserieResponseDto(referanse))
     }
 
+    @ProtectedWithClaims(issuer = "aad", claimMap = ["roles=faktureringskomponenten-skriv"])
+    @PutMapping("/{referanse}/mottaker")
+    fun oppdaterFakturaMottaker(
+        @RequestBody @Validated fakturamottakerRequestDto: FakturamottakerRequestDto,
+        @PathVariable("referanse", required = true) referanse: String,
+        bindingResult: BindingResult
+    ): ResponseEntity<Any> {
+        log.info("Mottatt spørsmål om endring av fakturamottaker for ${referanse}")
+
+        if (bindingResult.hasErrors()) {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(ProblemDetailFactory.of(bindingResult))
+        }
+
+        val fakturamottakerDto = fakturamottakerRequestDto.tilFakturamottakerDto
+        faktureringService.endreFakturaMottaker(referanse, fakturamottakerDto)
+
+
+        return ResponseEntity.ok().build()
```

Comment:
Hva skal skje hvis man ikke kan endre fakturamottaker? Skal vi returnere OK da og?

Reply:
Ja, for det er helt lov, hvis feks mottaker er den samme som allerede ligger inne, eller hvis man "er for sen" og den siste fakturaen er sendt ut.

Reply:
Tenker jeg da 🤷‍♀️ Eventuell feilmelding hadde bare blitt brukt for å se i loggen hva som har skjedd. Kan logge en log.info i FakturaserieService dersom det ikke oppdaterer.

Reply:
Skjønner

Reply:
Logger litt: [92efad6](https://github.com/navikt/faktureringskomponenten/pull/104/commits/92efad68e19d9bbe0087d8bad34c3f7951989d2c)